File: CS50/Problems/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //get num between 1 and 8.
    int n;
    do
    {
        n = get_int("Height: ");
      
    }
    while (n < 1 || n > 8);

    // rows
    for (int height = 1; height <= n; height++)
    {
        // spaces
        for (int row = n; row > height; row--)
        {
            printf(" ");

        }
        // left pyramid
        for (int x = 0; x < height; x++)
        {
            printf("#");

        }
        // middle
        for (int d = 0; d < 1; d++)
        {
            printf("  ");

        }
        //right pyramid
        for (int y = 0; y < height; y++)
        {
            printf("#");

        }
        printf("\n");

    }
}

File: CS50/Problems/sentimental-mario-more/mario.py
from cs50 import get_int
# height
N = 0
while N < 1 or N > 8:
    N = get_int("Height: ")

# rows
for i in range(1, N + 1, 1):
    # spaces
    for j in range(N, i, -1):
        print(" ", end="")
    # left    
    for q in range(0, i, 1):
        print("#", end="")
    # mid
    for w in range(1):
        print("  ", end="")
    # right
    for e in range(i):
        print("#", end="")
    print("")    